command/agentproxyshared/auth/auth.go

```diff
@@ -478,6 +511,8 @@ func (ah *AuthHandler) Run(ctx context.Context, am AuthMethod) error {
 		}
 
 		metrics.IncrCounter([]string{ah.metricsSignifier, "auth", "success"}, 1)
+		// Set authenticated when authentication succeeds
```

Comment:
We can delete this comment and the one below as well.

Reply:
fixed